azure-core/azure/core/pipeline/base.py

```diff
@@ -23,25 +23,19 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
-
-import json
 import logging
-import os.path
-try:
-    from urlparse import urlparse
-except ImportError:
-    from urllib.parse import urlparse
-import xml.etree.ElementTree as ET
+from typing import (TYPE_CHECKING, Generic, TypeVar, cast, IO, List, Union, Any, Mapping, Dict, Optional, # pylint: disable=unused-import
+                    Tuple, Callable, Iterator)
 from azure.core.pipeline import AbstractContextManager, PipelineRequest, PipelineResponse, PipelineContext
 from azure.core.pipeline.policies import HTTPPolicy, SansIOHTTPPolicy
-from typing import TYPE_CHECKING, Generic, TypeVar, cast, IO, List, Union, Any, Mapping, Dict, Optional, Tuple, Callable, Iterator  # pylint: disable=unused-import
 
 HTTPResponseType = TypeVar("HTTPResponseType")
 HTTPRequestType = TypeVar("HTTPRequestType")
 
 _LOGGER = logging.getLogger(__name__)
 
 
+#pylint: disable=too-few-public-methods
```

Comment:
14 times in this PR and 6 times already in the repo, another pylint config that might be worth considering changing overall